### ISSUE TITLE
Can't get rename to work

### ISSUE THREAD

**Issue by michaelkonecny** [hansec/ide-fortran#13]
Hi,

sorry to bother you. I noticed you added symbol renaming.

I just can't get it to work.

I've tried right clicking on the variable and selecting rename

![image](https://user-images.githubusercontent.com/4976024/57735594-6c4a4980-76a5-11e9-884a-5bbc98693351.png)

as well as Ctrl+Shift+P -> Nuclide Refactorizer: Rename, but neither seem to work.

Can you please just tell me how you invoke the command so I can debug further?

**Comment by hansec** [comment]
Oops, actually I don't think "rename" is supported yet by the [Atom LS interface](https://github.com/atom/atom-languageclient). I have only used this feature in VSCode. It was included on the README for this package by accident. The rename menu option doesn't work on Atom for me either and it doesn't look like Atom is sending a rename request to the server so it might be Nuclide specific.

**Comment by michaelkonecny** [comment]
Yeah, it [seems as though](https://github.com/facebookarchive/atom-ide-ui/issues/145) this was WIP in atom-ide-ui before Facebook dropped the project, so hopefully someone will take it up in the community project...
